Software/view/view.py

```diff
@@ -0,0 +1,56 @@
+from tkinter import Tk, Label
+
+class View:
+    def __init__(self, numberOfPlaces):
+        self.numberOfPlaces = numberOfPlaces;
```

Comment:
Another superfluous `;`, please remove it

Reply:
fixed